pkg/helm/chartproxy/proxy.go

```diff
@@ -91,13 +91,23 @@ func (p *proxy) IndexFile(onlyCompatible bool, namespace string) (*repo.IndexFil
 
 			for key, entry := range idxFile.Entries {
 				for i := len(entry) - 1; i >= 0; i-- {
+					if entry[i].Annotations == nil {
+						annotations := make(map[string]string, 0)
+						annotations["repositoryName"] = helmRepo.Name
```

Comment:
Move and rename this magic string to a constant:

```go
const RepositoryNameKey = "openshift.helm/repository-name"
```

The name `openshift.helm/repository-name` is only used internally, and could be modified later on.

Reply:
No longer used